.circleci/config.yml

```diff
@@ -7,17 +7,16 @@ orbs:
 jobs:
   install_and_update_dependencies:
     docker:
-    - image: cimg/python:3.10
+    - image: cimg/python:3.12
     working_directory: ~/repo
     steps:
     - checkout
     - restore_cache:
         keys:
-        - v3-dependencies-{{ checksum "Pipfile.lock" }}
-        - v3-dependencies-default
+        - v4-dependencies-{{ checksum "Pipfile.lock" }}
+        - v4-dependencies-default
```

Comment:
Not really sure how/why this happened, but I somehow saved a bad cache. I have not seen this before.

I'm not going to dwell on this too much - just going to invalidate and move on.